src/components/slide.js

```diff
@@ -95,11 +102,17 @@ const Slide = props => {
   return (
     <SlideContainer
       ref={slideRef}
-      backgroundColor={backgroundColor}
-      style={{
-        transform: `scale(${ratio})`,
-        transformOrigin: `${origin.x} ${origin.y}`
-      }}
+      backgroundColor={
+        window.location.search.includes('print') ? '#ffffff' : backgroundColor
```

Comment:
Could we get this from the `useUrlRouting` hook?

Reply:
Missed this, will get this done!